src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.model;
+
+import racingcar.util.RandomNumberUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+
+    public Cars(int numberOfCar) {
+        this.cars = initCars(numberOfCar);
+    }
+
+    public List<Car> initCars(int numberOfCar) {
+        List<Car> cars = new ArrayList<Car>();
+
+        for (int i = 0; i < numberOfCar; i++) {
+            cars.add(new Car());
+        }
+
+        return cars;
+    }
```

Comment:
```suggestion
    private Cars(List<Car> cars) {
        this.cars = cars;
    }
    
    public static of Cars(List<Car> cars) {
        return new Cars(cars);
    }
    
    public static of Cars(List<Car> cars) {
        List<Car> cars = new ArrayList<Car>();

        for (int i = 0; i < numberOfCar; i++) {
            cars.add(new Car());
        }

        return Cars.of(cars);
    }
```

이렇게 정적 생성 팩토리 메소드 패턴을 사용해보면 어떨까요?

Reply:
네 정적 생성 팩토리 메소드 패턴 적용해서 구현 해보겠습니다 !